cmd/juju/model/destroy.go

```diff
@@ -112,10 +122,15 @@ func (c *destroyCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.ModelCommandBase.SetFlags(f)
 	f.BoolVar(&c.assumeYes, "y", false, "Do not prompt for confirmation")
 	f.BoolVar(&c.assumeYes, "yes", false, "")
+	f.BoolVar(&c.destroyStorage, "destroy-storage", false, "Destroy all storage in the model")
+	f.BoolVar(&c.releaseStorage, "release-storage", false, "Release all storage from the model, and management of the controller, without destroying them")
```

Comment:
Is "storage" plural? To match "them"? Or is it a collective noun?
Maybe "storage instances" is better?